lib/trubl/base.rb

```diff
@@ -28,7 +28,9 @@ def missing_or_exception?(response)
           nil
         end
 
-        code && (400..600).include?(code)
+        is_missing_or_exception = code && (400..600).include?(code)
```

Comment:
Yes, @javidjamae and I talked about this as we worked on it. We don't quite understand the original author's reason or intention of such duplication. And since I think the idea of private method and methods sharing is conflict to some degree, I deferred the refactoring. @rottgoth do you have any suggestion to break this ice (sharing private method)?